test/support/page_objects.js

```diff
@@ -0,0 +1,74 @@
+
+import Common from './pages/common.js';
+import ConsolePage from './pages/console_page.js';
+import DashboardPage from './pages/dashboard_page.js';
+import DiscoverPage from './pages/discover_page.js';
+import HeaderPage from './pages/header_page.js';
+import SettingsPage from './pages/settings_page.js';
+import ShieldPage from './pages/shield_page.js';
+import VisualizePage from './pages/visualize_page.js';
+
+const common = new Common();
+const consolePage = new ConsolePage();
```

Comment:
What do you think about having page objects register themselves with the `PageObjects` service instead of having them explicitly listed here?

That way external code (like tests from plugins) could do `PageObjects.register(MyPluginPage)` and then test files could required the page objects that they need explicitly... 

Just a thought